src/main/resources/select-options-by-index.js

```diff
@@ -14,17 +14,25 @@
     return {disabledOptions: disabledOptionsIndexes};
   }
 
-  const previousSelectedIndex = select.selectedIndex;
-  var fireChangeEvent = false;
+  function getSelectedOptions(select) {
+    var result = []
+    for (var i = 0; i < select.options.length; i++) {
+      if (select.options[i].selected) {
+        result.push(i);
+      }
+    }
+    return result;
+  }
```

Comment:
This function may be rewritten in functional style:
```javascript
function getSelectedOptions(select) {
  return Array.from(select.options).map(option => option.selected);
  // it returns array of booleans, e.g. [false, true, false, true, false, false]
}
```

Reply:
I was wondering about navigator compatibility, I see that it's not compatible only with IE 11 (as almost always)